packages/ocamlmig/ocamlmig.5.2-20250129/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "2.0"
+synopsis: "Ocaml source code rewriting tool"
```

Comment:
```suggestion
synopsis: "OCaml source code rewriting tool"
```

fixing typo

Reply:
Thanks, it's fixed in the source repository now.